setup.cfg

```diff
@@ -59,10 +59,13 @@ install_requires =
     # SDL2 bindings. For Windows and macOS there are pre-built SDL2 binaries provided.
     pysdl2          <  2.0
     pysdl2-dll      ~= 2.0; sys_platform == "win32" or sys_platform == "darwin"
+
+[options.extras_require]
+joystick =
     # MIDI controller support.
```

Comment:
Ah, I was expecting that SDL2 is also going to be moved into the `joystick` option. However, I see now that it may not be necessary because PySDL2 doesn't really carry any dependencies so it should be cheap to keep. In this case though, would it not be better to rename the installation option from `joystick` to `midi`, as it would make its purpose clearer?